test/standalone/Bucket/restart.jl

```diff
@@ -0,0 +1,102 @@
+using Test
+import ClimaComms
+ClimaComms.@import_required_backends
+import ClimaLand
+import ClimaParams
+import SciMLBase
+import ClimaTimeSteppers
+import Dates
+import ClimaUtilities
+
+# First, run a simulation for 1 hour
```

Comment:
right now it runs for 3hr

Reply:
Thank you, I also updated the describe the test.